src/lib/mod.rs

```diff
@@ -1,14 +1,97 @@
 pub mod barcode_matching;
+pub mod bitenc;
 pub mod samples;
 
+use crate::bitenc::BitEnc;
+use lazy_static::lazy_static;
```

Comment:
It is no longer necessary to use `lazy_static` thanks to the addition of `std::sync::LazyLock`. If this was legacy I'd say just add it as a todo, but since it's new I'd say do it in this PR to avoid introducing a new unnecessary dependency.